assets/default-schema.graphql

```diff
@@ -1,3 +1,4 @@
+"A date string with format 'Y-m-d H:i:s', e.g. '2018-01-01 13:00:00'"
```

Comment:
According graphql spec it would be """A date string with format ...""" (triple double quote). Am i wrong?

Reply:
Both are allowed.